bapc/ingress.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: bapc
+  namespace: default
+spec:
+  rules:
+    - host: bapc.eu
+      http:
+        paths:
+        - backend:
+            serviceName: bapc-2020
+            servicePort: 80
+          path: /2020
+  tls:
+    - hosts:
+        - bapc.eu
+      secretName: ingress-ext-tls
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.ingress.kubernetes.io/temporal-redirect: https://2020.bapc.eu
```

Comment:
Moeten wij die redirect doen of doet de eigenaar van de domeinnaam dat?

Reply:
Hmm goeie vraag, ik zou het eerlijk gezegd niet zo snel weten. Als het niet werkt moet ik maar even achter de eigenaar van het domein aan.

Reply:
In afwachting daarop zou ik in deze file alleen de ingress voor `2020.bapc.eu` (bovenin) laten staan en de andere wijzigingen weghalen.